Description: when Sphinx crashes, show 10 stack frames (instead of a single one)
 Normally, when Sphinx crashes, it doesn't display full stack trace (as other
 Python application do by default), but only a single one; rest of the stack
 trace is stored into a temporary file. Such behaviour is undesired in some
 circumstances; e.g. is Sphinx crashes on a buildd, the full stack trace is
 definitely lost.
 .
 This patch mitigates the problem by increasing number of show stack frames to
 10.
Author: Jakub Wilk <jwilk@debian.org>
Forwarded: not-needed
Last-Update: 2011-05-18

--- a/sphinx/cmdline.py
+++ b/sphinx/cmdline.py
@@ -212,7 +212,7 @@
                 print >>error, terminal_safe(unicode(err))
             else:
                 print >>error, red('Exception occurred:')
-                print >>error, format_exception_cut_frames().rstrip()
+                print >>error, format_exception_cut_frames(10).rstrip()
                 tbpath = save_traceback()
                 print >>error, red('The full traceback has been saved '
                                    'in %s, if you want to report the '
